src/lib/clk.c

```diff
@@ -115,6 +115,19 @@ uint64_t clock_ms_to_ticks(int clock, uint64_t ms)
 	return ticks;
 }
 
+uint64_t clock_sample_ticks(int clock, uint32_t sample_rate)
```

Comment:
inline comments here explaining usage. I'm assuming we get number of ticks per sample ?

Reply:
yep, let me add inline comments.